src/model/add-dots.ts

```diff
@@ -0,0 +1,56 @@
+// Wraps the last dot in a footer
+const transformLastDot = (element: string): string => {
```

Comment:
Maybe `transformDots`? It's more descriptive and means the same function can be used in the body